src/pages/Swap/index.tsx

```diff
@@ -53,7 +53,9 @@ export default function Swap() {
     useCurrency(loadedUrlParams?.inputCurrencyId),
     useCurrency(loadedUrlParams?.outputCurrencyId)
   ]
-  const [dismissTokenWarning, setDismissTokenWarning] = useState<boolean>(false)
+  const [dismissTokenWarning, setDismissTokenWarning] = useState<boolean>(
```

Comment:
you're changing their file here

Reply:
i would leave this as default and just change the yarn start command to work with your env

Reply:
Yes, I hate that I changed their file, but otherwise I need to modify the Swap component

Why do you say I can do it in YARN? maybe you refer to the env var, but the issue is that this env var didn't exist before